easybuild/easyconfigs/p/pymatgen/pymatgen-4.1.1-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = "PythonPackage"
+
+name = 'pymatgen'
+version = '4.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/pymatgen'
+description = """Python Materials Genomics is a robust materials analysis code that defines core object
+ representations for structures and molecules with support for many electronic structure codes."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('spglib-python', '1.9.4.2', versionsuffix),
```

Comment:
@migueldiascosta OK, but then what about these issues (with the `pymatgen` module loaded)? The first one looks like a problem with `numpy`, the 2nd with `spglib`? Is this working for you?

```
>>> from pymatgen import Molecule
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/user/home/gent/vsc400/vsc40023/eb_phanpyscratch/SL6/sandybridge/software/pymatgen/4.1.1-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/pymatgen-4.1.1-py2.7-linux-x86_64.egg/pymatgen/__init__.py", line 13, in <module>
    import spglib
  File "/user/home/gent/vsc400/vsc40023/eb_phanpyscratch/SL6/sandybridge/software/spglib-python/1.9.4.2-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/spglib/__init__.py", line 35, in <module>
    from .spglib import (get_version,
  File "/user/home/gent/vsc400/vsc40023/eb_phanpyscratch/SL6/sandybridge/software/spglib-python/1.9.4.2-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/spglib/spglib.py", line 512, in <module>
    is_shift=np.zeros(3, dtype='intc')):
AttributeError: 'module' object has no attribute 'zeros'
```
```
>>> import pymatgen.symmetry.analyzer
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/user/home/gent/vsc400/vsc40023/eb_phanpyscratch/SL6/sandybridge/software/pymatgen/4.1.1-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/pymatgen-4.1.1-py2.7-linux-x86_64.egg/pymatgen/__init__.py", line 13, in <module>
    import spglib
  File "/user/home/gent/vsc400/vsc40023/eb_phanpyscratch/SL6/sandybridge/software/spglib-python/1.9.4.2-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/spglib/__init__.py", line 35, in <module>
    from .spglib import (get_version,
  File "/user/home/gent/vsc400/vsc40023/eb_phanpyscratch/SL6/sandybridge/software/spglib-python/1.9.4.2-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/spglib/spglib.py", line 35, in <module>
    from . import _spglib as spg
ImportError: cannot import name _spglib
>>> 
```